### ISSUE TITLE
Focus already open NVDA settings dialog after closure of the Error dialog upon attempting to open another NVDA settings dialog

### ISSUE THREAD

**Issue by bhavyashah** [nvaccess/nvda#6371]
Steps to reproduce
Open any NVDA settings dialog, for instance press NVDA+Ctrl+V to open the Voice Settings dialog.
Now, Alt+Tab away to another window, and attempt to open another or the same NVDA settings dialog.
You should be presented with an Error dialog asking to close the already open NVDA settings dialog first.  Press the Ok button.
Expected/Requested Result: Focus should move to the window of the already open NVDA settings dialog, due to the following reasons:
* If the user issued the same settings dialog command, his intent is to work with that same settings dialog. The proposed result fulfills this scenario's requirements.
* If the user wishes to open another NVDA settings dialog, his intent is to close the already open dialog before being able to open another one. This scenario's requirements are also fulfilled by the proposed result as directly being placed on the already open settings dialog allows the user to quickly close it and continue opening a new one.
Actual Result: After the user is simply notified that an existing NVDA settings dialog needs to be closed, he is placed back in the window where his focus originally was. This does not serve any purpose, as the user will would require to then navigate to the open settings dialog manually and then perform subsequently desired actions.

Test Environment
Windows 8.1 Professional 64 bit
NVDA version 2016.3

**Comment by jcsteh** [comment]
Duplicate of #5383, which you also reported. :)

**Comment by bhavyashah** [comment]
Oh, sincerest apologies! I often have ideas and things for NVDA in my
head, and sometimes forget that I have already reported some of them.
Sorry for the confusion. :)

On 9/13/16, James Teh <notifications@github.com> wrote:
> Duplicate of #5383, which you also reported. :)
>
> --
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/nvaccess/nvda/issues/6371#issuecomment-246581730


-- 
Warm Regards
Bhavya Shah
Using NVDA (Non Visual Desktop Access) free and open source screen
reader for Microsoft Windows
To download a copy of the free screen reader NVDA, please visit
http://www.nvaccess.org/
Using Google Talkback on Motorolla G second generation Lollipop 5.0.2
Reach me through the following means:
Mobile: +91 7506221750
E-mail id: bhavya.shah125@gmail.com
Skype id : bhavya.09
